pkg/config/config_template.yaml

```
@@ -973,6 +973,7 @@ api_key:
   ## @param batch_wait - integer - optional - default: 5
   ## @env DD_LOGS_CONFIG_BATCH_WAIT - integer - optional - default: 5
   ## The maximum time the Datadog Agent waits to fill each batch of logs before sending.
+  ## The maximum batch wait in seconds.
```

What do you think of 

```suggestion
  ## The maximum time (in seconds) the Datadog Agent waits to fill each batch of logs before sending.
```
or
```suggestion
  ## The maximum time the Datadog Agent waits to fill each batch of logs before sending (in seconds).
```
?

---

Thank you @pgimalac, that's a much better solution!